crates/iota/tests/cli_tests.rs

```diff
@@ -4578,3 +4578,18 @@ async fn test_call_command_emit_args() -> Result<(), anyhow::Error> {
 
     Ok(())
 }
+
+#[sim_test]
+async fn test_ptb_dev_inspect() -> Result<(), anyhow::Error> {
+    let mut test_cluster = TestClusterBuilder::new().build().await;
+    let context = &mut test_cluster.wallet;
+
+    let publish_ptb_string = r#"
+        --assign hello_option "some('Hello')" \
+        --move-call std::option::borrow "<std::string::String>" hello_option \
+        --dev-inspect
+        "#;
+    let args = shlex::split(publish_ptb_string).unwrap();
+    iota::client_ptb::ptb::PTB { args }.execute(context).await?;
```

Comment:
Maybe add a check that the output contains `IOTA TypeTag: IotaTypeTag("0x1::string::String")` and `Bytes: [5, 72, 101, 108, 108, 111]`?

Reply:
The output is just printed and not returned, is there a good way to check this?

Reply:
Ah, I see, annoying. We can refactor `execute` a bit so it returns a String and then we print it. But that would be another PR.